Cryptomator 1.14.12 AppImage does not show "password" page on Linux

<!--

Do you want to ask a QUESTION? Are you looking for SUPPORT?
We're happy to help you via our support channels! Please read: https://github.com/cryptomator/cryptomator/blob/develop/SUPPORT.md

By filing an issue, you are expected to comply with our code of conduct: https://github.com/cryptomator/cryptomator/blob/develop/.github/CODE_OF_CONDUCT.md

Of course, we also expect you to search for existing similar issues first! ;) https://github.com/cryptomator/cryptomator/issues?q=

-->

### Description

Cryptomator does not show the password page on Linux, making it impossible to unlock the vault. It worked fine in 1.14.11.

### System Setup

* Operating system and version: Fedora 30 KDE
* Cryptomator version: 1.14.12
* Drive: Fuse

### Steps to Reproduce

1. Attempt to open password page on Linux by clicking the vault
2. Only the robot logo shows
3. You can not unlock the vault

#### Expected Behavior

Being able to unlock the vault

#### Actual Behavior

The view doesn't change, therefore you can't type in the password.

#### Reproducibility

Always

### Additional Information

[Any additional information, log files, screenshots, configuration, or data that might be necessary to reproduce the issue.]

<!--

If you want to add the log file or screenshots, please add them as attachments. If your log file seems empty and doesn't show any errors, you may enable the debug mode first. Here is how to do that: https://community.cryptomator.org/t/how-do-i-enable-debug-mode/36

Then reproduce the problem to ensure all important information is contained in there. You may use test data or redact sensitive information from the log file.

Log file location:
- Windows: %appdata%/Cryptomator
- macOS: ~/Library/Logs/Cryptomator
- Linux: ~/.local/share/Cryptomator/logs

-->

`/usr/bin/AppImageLauncher: /lib64/libcurl.so.4: no version information available (required by /usr/bin/../lib/x86_64-linux-gnu/appimagelauncher/libappimageupdate.so)

(process:589): GLib-CRITICAL **: 09:58:32.474: g_file_test: assertion 'filename != NULL' failed
Icon theme "elementary" not found.
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=on -Dswing.aatext=true -Dswing.defaultlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel -Dswing.crossplatformlaf=com.sun.java.swing.plaf.gtk.GTKLookAndFeel
09:58:37.081 [main] DEBUG org.cryptomator.common.Environment - user.language: en
09:58:37.097 [main] DEBUG org.cryptomator.common.Environment - user.region: null
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - logback.configurationFile: null
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - cryptomator.settingsPath: ~/.config/Cryptomator/settings.json:~/.Cryptomator/settings.json
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - cryptomator.ipcPortPath: ~/.config/Cryptomator/ipcPort.bin:~/.Cryptomator/ipcPort.bin
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - cryptomator.keychainPath: null
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - cryptomator.logDir: ~/.local/share/Cryptomator/logs
09:58:37.098 [main] DEBUG org.cryptomator.common.Environment - cryptomator.mountPointsDir: ~/.local/share/Cryptomator/mnt
09:58:37.646 [main] DEBUG org.cryptomator.common.settings.SettingsProvider - Attempting to load settings from /home/blah/.config/Cryptomator/settings.json
09:58:37.776 [main] INFO org.cryptomator.common.settings.SettingsProvider - Settings loaded from /home/blah/.config/Cryptomator/settings.json
09:58:38.484 [main] INFO org.cryptomator.launcher.FileOpenRequestHandler - Unable to setOpenFileHandler, probably not supported on this OS.
09:58:38.525 [main] INFO  org.cryptomator.launcher.Cryptomator - Starting Cryptomator 1.4.12 on Linux 5.1.19-300.fc30.x86_64 (amd64)
09:58:38.737 [main] INFO  o.c.launcher.IpcProtocolImpl - Received launch args: 
09:58:39.282 [JavaFX Application Thread] INFO  org.cryptomator.launcher.Cryptomator - JavaFX application started.
09:58:39.900 [JavaFX Application Thread] ERROR o.f.secret.simple.SimpleCollection - org.freedesktop.dbus.exceptions.DBusException: Failed to connect to bus Failed to auth
java.io.IOException: Failed to auth
        at org.freedesktop.dbus.connections.Transport.connect(Transport.java:150)
        at org.freedesktop.dbus.connections.Transport.<init>(Transport.java:70)
        at org.freedesktop.dbus.connections.AbstractConnection.<init>(AbstractConnection.java:149)
        at org.freedesktop.dbus.connections.impl.DBusConnection.<init>(DBusConnection.java:284)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:122)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:142)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:200)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:157)
        at org.freedesktop.secret.TransportEncryption.<init>(TransportEncryption.java:36)
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:112)
        at org.freedesktop.secret.simple.SimpleCollection.<init>(SimpleCollection.java:44)
        at org.cryptomator.keychain.GnomeKeyringAccessImpl.<init>(GnomeKeyringAccessImpl.java:18)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.cryptomator.keychain.LinuxKeychainTester.getSecretService(LinuxKeychainTester.java:22)
        at org.cryptomator.keychain.LinuxSecretServiceAccess.<init>(LinuxSecretServiceAccess.java:14)
        at org.cryptomator.keychain.LinuxSecretServiceAccess_Factory.get(LinuxSecretServiceAccess_Factory.java:16)
        at org.cryptomator.keychain.LinuxSecretServiceAccess_Factory.get(LinuxSecretServiceAccess_Factory.java:6)
        at org.cryptomator.keychain.KeychainModule_ProvideKeychainAccessStrategiesFactory.get(KeychainModule_ProvideKeychainAccessStrategiesFactory.java:37)
        at org.cryptomator.keychain.KeychainModule_ProvideKeychainAccessStrategiesFactory.get(KeychainModule_ProvideKeychainAccessStrategiesFactory.java:9)
        at dagger.internal.SetFactory.get(SetFactory.java:119)
        at dagger.internal.SetFactory.get(SetFactory.java:37)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:29)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:10)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:33)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:10)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:77)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:17)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:26)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:8)
        at org.cryptomator.ui.controllers.ViewControllerLoader.constructController(ViewControllerLoader.java:48)
        at javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:938)
        at javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:980)
        at javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:227)
        at javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:752)
        at javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2722)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2552)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2450)
        at org.cryptomator.ui.controllers.ViewControllerLoader.load(ViewControllerLoader.java:36)
        at org.cryptomator.launcher.Cryptomator$MainApp.start(Cryptomator.java:109)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:846)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:455)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
        at java.base/java.lang.Thread.run(Thread.java:834)
09:58:39.902 [JavaFX Application Thread] ERROR o.c.keychain.GnomeKeyringAccessImpl - D-Bus reports a problem.
java.io.IOException: org.freedesktop.dbus.exceptions.DBusException: Failed to connect to bus Failed to auth
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:126)
        at org.freedesktop.secret.simple.SimpleCollection.<init>(SimpleCollection.java:44)
        at org.cryptomator.keychain.GnomeKeyringAccessImpl.<init>(GnomeKeyringAccessImpl.java:18)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.cryptomator.keychain.LinuxKeychainTester.getSecretService(LinuxKeychainTester.java:22)
        at org.cryptomator.keychain.LinuxSecretServiceAccess.<init>(LinuxSecretServiceAccess.java:14)
        at org.cryptomator.keychain.LinuxSecretServiceAccess_Factory.get(LinuxSecretServiceAccess_Factory.java:16)
        at org.cryptomator.keychain.LinuxSecretServiceAccess_Factory.get(LinuxSecretServiceAccess_Factory.java:6)
        at org.cryptomator.keychain.KeychainModule_ProvideKeychainAccessStrategiesFactory.get(KeychainModule_ProvideKeychainAccessStrategiesFactory.java:37)
        at org.cryptomator.keychain.KeychainModule_ProvideKeychainAccessStrategiesFactory.get(KeychainModule_ProvideKeychainAccessStrategiesFactory.java:9)
        at dagger.internal.SetFactory.get(SetFactory.java:119)
        at dagger.internal.SetFactory.get(SetFactory.java:37)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:29)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:10)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:33)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:10)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:77)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:17)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:26)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:8)
        at org.cryptomator.ui.controllers.ViewControllerLoader.constructController(ViewControllerLoader.java:48)
        at javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:938)
        at javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:980)
        at javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:227)
        at javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:752)
        at javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2722)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2552)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2450)
        at org.cryptomator.ui.controllers.ViewControllerLoader.load(ViewControllerLoader.java:36)
        at org.cryptomator.launcher.Cryptomator$MainApp.start(Cryptomator.java:109)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:846)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:455)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.io.IOException: Failed to auth
        at org.freedesktop.dbus.connections.Transport.connect(Transport.java:150)
        at org.freedesktop.dbus.connections.Transport.<init>(Transport.java:70)
        at org.freedesktop.dbus.connections.AbstractConnection.<init>(AbstractConnection.java:149)
        at org.freedesktop.dbus.connections.impl.DBusConnection.<init>(DBusConnection.java:284)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:122)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:142)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:200)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:157)
        at org.freedesktop.secret.TransportEncryption.<init>(TransportEncryption.java:36)
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:112)
        ... 43 common frames omitted
09:58:39.945 [JavaFX Application Thread] ERROR o.f.secret.simple.SimpleCollection - org.freedesktop.dbus.exceptions.DBusException: Failed to connect to bus Failed to auth
java.io.IOException: Failed to auth
        at org.freedesktop.dbus.connections.Transport.connect(Transport.java:150)
        at org.freedesktop.dbus.connections.Transport.<init>(Transport.java:70)
        at org.freedesktop.dbus.connections.AbstractConnection.<init>(AbstractConnection.java:149)
        at org.freedesktop.dbus.connections.impl.DBusConnection.<init>(DBusConnection.java:284)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:122)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:142)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:200)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:157)
        at org.freedesktop.secret.TransportEncryption.<init>(TransportEncryption.java:36)
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:112)
        at org.freedesktop.secret.simple.SimpleCollection.<init>(SimpleCollection.java:44)
        at org.cryptomator.keychain.GnomeKeyringAccessImpl.<init>(GnomeKeyringAccessImpl.java:18)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.cryptomator.keychain.LinuxKeychainTester.getSecretService(LinuxKeychainTester.java:22)
        at org.cryptomator.keychain.LinuxSecretServiceAccess.isSupported(LinuxSecretServiceAccess.java:19)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
        at java.base/java.util.HashMap$KeySpliterator.tryAdvance(HashMap.java:1624)
        at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
        at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
        at org.cryptomator.keychain.KeychainModule.provideSupportedKeychain(KeychainModule.java:40)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.provideSupportedKeychain(KeychainModule_ProvideSupportedKeychainFactory.java:42)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:29)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:10)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:33)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:10)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:77)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:17)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:26)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:8)
        at org.cryptomator.ui.controllers.ViewControllerLoader.constructController(ViewControllerLoader.java:48)
        at javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:938)
        at javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:980)
        at javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:227)
        at javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:752)
        at javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2722)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2552)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2450)
        at org.cryptomator.ui.controllers.ViewControllerLoader.load(ViewControllerLoader.java:36)
        at org.cryptomator.launcher.Cryptomator$MainApp.start(Cryptomator.java:109)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:846)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:455)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
        at java.base/java.lang.Thread.run(Thread.java:834)
09:58:39.946 [JavaFX Application Thread] ERROR o.c.keychain.GnomeKeyringAccessImpl - D-Bus reports a problem.
java.io.IOException: org.freedesktop.dbus.exceptions.DBusException: Failed to connect to bus Failed to auth
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:126)
        at org.freedesktop.secret.simple.SimpleCollection.<init>(SimpleCollection.java:44)
        at org.cryptomator.keychain.GnomeKeyringAccessImpl.<init>(GnomeKeyringAccessImpl.java:18)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
        at org.cryptomator.keychain.LinuxKeychainTester.getSecretService(LinuxKeychainTester.java:22)
        at org.cryptomator.keychain.LinuxSecretServiceAccess.isSupported(LinuxSecretServiceAccess.java:19)
        at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:176)
        at java.base/java.util.HashMap$KeySpliterator.tryAdvance(HashMap.java:1624)
        at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:127)
        at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:502)
        at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:488)
        at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474)
        at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
        at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
        at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:543)
        at org.cryptomator.keychain.KeychainModule.provideSupportedKeychain(KeychainModule.java:40)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.provideSupportedKeychain(KeychainModule_ProvideSupportedKeychainFactory.java:42)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:29)
        at org.cryptomator.keychain.KeychainModule_ProvideSupportedKeychainFactory.get(KeychainModule_ProvideSupportedKeychainFactory.java:10)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:33)
        at org.cryptomator.ui.model.AutoUnlocker_Factory.get(AutoUnlocker_Factory.java:10)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:77)
        at org.cryptomator.ui.controllers.MainController_Factory.get(MainController_Factory.java:17)
        at dagger.internal.DoubleCheck.get(DoubleCheck.java:47)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:26)
        at org.cryptomator.ui.controllers.ViewControllerModule_ProvideMainControllerFactory.get(ViewControllerModule_ProvideMainControllerFactory.java:8)
        at org.cryptomator.ui.controllers.ViewControllerLoader.constructController(ViewControllerLoader.java:48)
        at javafx.fxml.FXMLLoader$ValueElement.processAttribute(FXMLLoader.java:938)
        at javafx.fxml.FXMLLoader$InstanceDeclarationElement.processAttribute(FXMLLoader.java:980)
        at javafx.fxml.FXMLLoader$Element.processStartElement(FXMLLoader.java:227)
        at javafx.fxml.FXMLLoader$ValueElement.processStartElement(FXMLLoader.java:752)
        at javafx.fxml.FXMLLoader.processStartElement(FXMLLoader.java:2722)
        at javafx.fxml.FXMLLoader.loadImpl(FXMLLoader.java:2552)
        at javafx.fxml.FXMLLoader.load(FXMLLoader.java:2450)
        at org.cryptomator.ui.controllers.ViewControllerLoader.load(ViewControllerLoader.java:36)
        at org.cryptomator.launcher.Cryptomator$MainApp.start(Cryptomator.java:109)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$9(LauncherImpl.java:846)
        at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$12(PlatformImpl.java:455)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$10(PlatformImpl.java:428)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at com.sun.javafx.application.PlatformImpl.lambda$runLater$11(PlatformImpl.java:427)
        at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:96)
        at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
        at com.sun.glass.ui.gtk.GtkApplication.lambda$runLoop$11(GtkApplication.java:277)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.io.IOException: Failed to auth
        at org.freedesktop.dbus.connections.Transport.connect(Transport.java:150)
        at org.freedesktop.dbus.connections.Transport.<init>(Transport.java:70)
        at org.freedesktop.dbus.connections.AbstractConnection.<init>(AbstractConnection.java:149)
        at org.freedesktop.dbus.connections.impl.DBusConnection.<init>(DBusConnection.java:284)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:122)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:142)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:200)
        at org.freedesktop.dbus.connections.impl.DBusConnection.getConnection(DBusConnection.java:157)
        at org.freedesktop.secret.TransportEncryption.<init>(TransportEncryption.java:36)
        at org.freedesktop.secret.simple.SimpleCollection.init(SimpleCollection.java:112)
        ... 48 common frames omitted
09:58:40.003 [JavaFX Application Thread] INFO  o.c.ui.controllers.MainController - Unable to setPreferencesHandler, probably not supported on this OS.
09:58:43.141 [JavaFX Application Thread] INFO  o.c.ui.controllers.WelcomeController - Current version: 1.4.12, lastest version: 1.4.11
`

--- a/appdir-build.sh
+++ b/appdir-build.sh
@@ -26,7 +26,7 @@ BUILD_VERSION=`cat libs/version.txt`
     --name Cryptomator \
     --version ${BUILD_VERSION} \
     --module-path ${JAVA_HOME}/jmods \
-    --add-modules java.base,java.logging,java.xml,java.sql,java.management,java.security.sasl,java.naming,java.datatransfer,java.security.jgss,java.rmi,java.scripting,java.prefs,java.desktop,jdk.unsupported \
+    --add-modules java.base,java.logging,java.xml,java.sql,java.management,java.security.sasl,java.naming,java.datatransfer,java.security.jgss,java.rmi,java.scripting,java.prefs,java.desktop,jdk.security.auth,jdk.unsupported \
     --strip-native-commands
 
 # archive app dir

